### DISCUSSION TITLE
request entity too large

### DISCUSSION THREAD

**Discussion by AhmedHashem2104** [adonisjs/core#2485] [Help]
{
  "message": "E_REQUEST_ENTITY_TOO_LARGE: request entity too large",
  "stack": "Exception: E_REQUEST_ENTITY_TOO_LARGE: request entity too large\n    at Multipart.validateProcessedBytes (E:\\Ahmed Hashem\\Own Projects\\AdsPrivateApp\\node_modules\\@adonisjs\\bodyparser\\build\\src\\Multipart\\index.js:82:20)\n    at PassThrough.<anonymous> (E:\\Ahmed Hashem\\Own Projects\\AdsPrivateApp\\node_modules\\@adonisjs\\bodyparser\\build\\src\\Multipart\\index.js:131:36)\n    at PassThrough.emit (events.js:327:22)\n    at addChunk (internal/streams/readable.js:309:12)\n    at readableAddChunk (internal/streams/readable.js:284:9)\n    at PassThrough.Readable.push (internal/streams/readable.js:223:10)\n    at PassThrough.Transform.push (internal/streams/transform.js:166:32)\n    at PassThrough.afterTransform (internal/streams/transform.js:101:10)\n    at PassThrough._transform (internal/streams/passthrough.js:46:3)\n    at PassThrough.Transform._read (internal/streams/transform.js:205:10)\n    at PassThrough.Transform._write (internal/streams/transform.js:193:12)\n    at writeOrBuffer (internal/streams/writable.js:358:12)\n    at PassThrough.Writable.write (internal/streams/writable.js:303:10)\n    at Form.onParsePartData (E:\\Ahmed Hashem\\Own Projects\\AdsPrivateApp\\node_modules\\multiparty\\index.js:479:43)\n    at Form._write (E:\\Ahmed Hashem\\Own Projects\\AdsPrivateApp\\node_modules\\multiparty\\index.js:426:10)\n    at writeOrBuffer (internal/streams/writable.js:358:12)",
  "code": "E_REQUEST_ENTITY_TOO_LARGE"
}
i get this json from uploading video 22mb, how to solve this issue ?

**Comment by thetutlage** [comment]
Please check the `config/bodyparser.ts` file and increase the `maxLimit` for multipart requests

**Reply by AhmedHashem2104** [reply]
it solved, thanks for you
